templates/webapi/test/link_context.html.ep

```diff
@@ -0,0 +1,21 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+<div class="container">
+        <p>
+            Test-Module:
+                <code><%= $contextpath %></code>
+        </p>
+        <textarea class='code' id='script'><%= $context %></textarea>
+
+        <script type="text/javascript">
+         var editor = CodeMirror.fromTextArea(document.getElementById("script"), {
+             lineNumbers: true,
+             readOnly: true,
+             lineWrapping: true
+         });
+        </script>
+</div>
```

Comment:
Likely the entire template should not be duplicated.

Reply:
i changed only the part that it introduced the editor as there are still some minor things that are different